types/office-js/index.d.ts

```diff
@@ -163,78 +289,101 @@ declare namespace Office {
          */
         name: string;
     }
+    /**
+     * Provides objects and methods that you can use to create and manipulate UI components, such as dialog boxes, in your Office Add-ins. 
+     */
     export interface UI {
         /**
-        * Displays a dialog to show or collect information from the user or to facilitate Web navigation.
-        * @param startAddress Accepts the initial HTTPS Url that opens in the dialog.
-        */
-        displayDialogAsync(startAddress: string): void;
-        /**
-        * Displays a dialog to show or collect information from the user or to facilitate Web navigation.
-        * @param startAddress Accepts the initial HTTPS Url that opens in the dialog.
-        * @param options Optional. Accepts a DialogOptions object to define dialog behaviors.
-        */
-        displayDialogAsync(startAddress: string, options: DialogOptions): void;
-        /**
-        * Displays a dialog to show or collect information from the user or to facilitate Web navigation.
-        * @param startAddress Accepts the initial HTTPS Url that opens in the dialog.
-        * @param callback Optional. Accepts a callback method to handle the dialog creation attempt.
-        */
-        displayDialogAsync(startAddress: string, callback: (result: AsyncResult) => void): void;
-        /**
-        * Displays a dialog to show or collect information from the user or to facilitate Web navigation.
-        * @param startAddress Accepts the initial HTTPS Url that opens in the dialog.
-        * @param options Optional. Accepts a DialogOptions object to define dialog behaviors.
-        * @param callback Optional. Accepts a callback method to handle the dialog creation attempt.
+        * Displays a dialog to show or collect information from the user or to facilitate Web navigation. 
+		*         
+        * @remarks
+        * Hosts: Word, Excel, Outlook, PowerPoint
+        * 
+        * Requirement sets: DialogApi, Mailbox 1.4
```

Comment:
instead of comma should it be `OR` (or is it AND?)
I think the point to convey to developers is that they can check for EITHER of these two API sets.
For DialogApi, please list its version number as well, 1.1